digital_image_processing/change_contrast.py

```diff
@@ -17,7 +17,7 @@ def change_contrast(img: Image, level: int) -> Image:
     """
     factor = (259 * (level + 255)) / (255 * (259 - level))
 
-    def contrast(c: int) -> int:
+    def contrast(c: int) -> float:

```

Comment:
Should __c__ be an __int__ or a __float__?

Please add "ready for review" when you are satisfied (and the Travis tests pass).

Reply:
`c` should be `int` it is actually individual color value of a pixel.
i.e
`[(c1, c2, c3), (c1, c2, c3)]`